src/stories/containers/Finances/useFinances.tsx

```diff
@@ -7,42 +7,47 @@ import { useCardChartOverview } from './components/SectionPages/CardChartOvervie
 import { useDelegateExpenseTrendFinances } from './components/SectionPages/DelegateExpenseTrendFinances/useDelegateExpenseTrendFinances';
 import { useMakerDAOExpenseMetrics } from './components/SectionPages/MakerDAOExpenseMetrics/useMakerDAOExpenseMetrics';
 import { prefixToRemove, removePrefix } from './utils/utils';
+import type { BudgetAnalytic } from '@ses/core/models/interfaces/analytic';
 import type { Budget } from '@ses/core/models/interfaces/budget';
 
-export const useFinances = (budgets: Budget[]) => {
-  const cardsNavigation: Budget[] = budgets.filter((budget) => budget.parentId === null);
+export const useFinances = (budgets: Budget[], initialYear: string, budgetsAnalytics: BudgetAnalytic[]) => {
+  console.log(budgetsAnalytics); // temporary
   const { isLight } = useThemeContext();
   const colors: string[] = ['#F99374', '#447AFB', '#2DC1B1'];
   const colorsDark: string[] = ['#F77249', '#447AFB', '#1AAB9B'];
   const router = useRouter();
 
-  const cardsNavigationInformation = cardsNavigation.map((item, index) => ({
-    image: item.image || '',
-    title: removePrefix(item.name, prefixToRemove),
-    description: item.description || 'Finances of the core governance constructs described in the Maker Atlas.',
-    href: `finances/${item.codePath}`,
-    totalDai: 132345,
-    valueDai: 12345,
-    color: isLight ? colors[index] : colorsDark[index],
-  }));
+  const cardsNavigationInformation = budgets.map((item, index) => {
+    const budgetMetric = budgetsAnalytics.find((ba) => ba.codePath === item.codePath);
+
+    return {
+      image: item.image || '',
+      title: removePrefix(item.name, prefixToRemove),
+      description: item.description || 'Finances of the core governance constructs described in the Maker Atlas.',
+      href: `/finances/${item.codePath.replace('atlas/', '')}`,
```

Comment:
Please, generate the `href` using `siteRoutes` instead to avoid issues in the future once the url changes

Reply:
Done! check: https://github.com/makerdao-ses/ecosystem-dashboard/pull/1117/commits/dcb1808c34b8389cc547be03221a06f17d27209d